torchao/prototype/float8nocompile/float8nocompile_linear.py

```diff
@@ -80,15 +80,19 @@ def forward(self, input: torch.Tensor) -> torch.Tensor:
         return output
 
     @classmethod
-    def from_float(cls, mod, kernel_algo: KernelAlgorithm = KernelAlgorithm.ATOMIC_MAX):
+    def from_float(
+        cls,
+        mod,
+        config: Float8LinearConfig,
```

Comment:
add comment specifying that only default config is supported and any non-defaults are silently ignored

Reply:
done, updated docstring as well